### DISCUSSION TITLE
React Table Column Resize Dynamically

### DISCUSSION THREAD

**Discussion by mohit-llp** [TanStack/table#2178] [General]
I just want to show column size dynamically i.e. resize as per as text length 
show scroll when reached maximum table Width.

See below image: -
![image](https://user-images.githubusercontent.com/44094302/79446041-3f13c300-7ffb-11ea-84b3-18d847928333.png)
here message column text is broken, here I want a scroll

Help me out this one!
thanks!

**Comment by mohit-llp** [comment]
Is it possible to add (text...) whenever reached to max column length?

**Comment by tannerlinsley** [comment]
You have to add a max-width in your cell content and use the proper `white-space` css property. Converting this to a discussion as it is not a bug, but a user implementation question.
